cluster/addons/metadata-proxy/gce/metadata-proxy.yaml

```diff
@@ -38,7 +38,7 @@ spec:
       dnsPolicy: Default
       containers:
       - name: metadata-proxy
-        image: gcr.io/google_containers/metadata-proxy:v0.1.6
+        image: gcr.io/google_containers/metadata-proxy:v0.1.7
```

Comment:
@ihmccreery - what changed in this version?

Reply:
https://github.com/GoogleCloudPlatform/k8s-metadata-proxy/pull/13

and change base image.